src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -175,7 +175,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv6 loopback interface.");
```

Comment:
And `UvException.Message`. That would leave out the stack trace. Sometimes libuv uses different error codes on different platforms, so having the message could be useful.